src/main/java/sk/tomsik68/mclauncher/impl/login/yggdrasil/YDServiceAuthenticationException.java

```diff
@@ -1,25 +1,32 @@
 package sk.tomsik68.mclauncher.impl.login.yggdrasil;
 
+import com.sun.istack.internal.Nullable;
+
 public class YDServiceAuthenticationException extends Exception {
-	
-	/**
-	 * 
-	 */
-	private static final long serialVersionUID = -7648783527451750219L;

```

Comment:
Do you know what it is for?

Reply:
I see that is `Serialized` object, but i don't understand why you use it. Can you help me?

Reply:
It was probably there because it extends `Exception`, I don't see any problem with removing it.